src/main/java/com/github/onsdigital/babbage/api/endpoint/rss/service/RssService.java

```diff
@@ -109,6 +109,34 @@ public BabbageRssResponse getDataListFeedResponse(HttpServletRequest request) {
         return writeToResponse(listFeed(request));
     }
 
+    /**
+     * @return a {@link SyndFeed} for the Release Calendar RSS feed defined by the requested parameters.
+     */
+    public SyndFeed getReleaseCalendarFeed(HttpServletRequest request, SearchQueries searchQueries) {
+        Optional<SearchResult> results = search(searchQueries);
+        StringBuilder title = new StringBuilder(calendarTitle(request));
+        String query;
+
+        if ((query = request.getParameter("query")) != null) {
+            title.append(" ").append(String.format(calendarTitleTwo, query));
+        }
+
+        return new SyndFeedBuilder()
+                .type(rssType)
+                .link(request.getRequestURL().toString())
+                .category(title.toString())
+                .entries(searchResultsToSyndEntries(results))
+                .title(title.toString())
+                .build();
+    }
+
+    /**
+     * @return a {@link BabbageRssResponse} for the Release Calendar RSS feed defined by the requested parameters.
+     */
+    public BabbageRssResponse getReleaseCalendarFeedResponse(HttpServletRequest request, SearchQueries searchQueries) {
+        return writeToResponse(getReleaseCalendarFeed(request, searchQueries));
+    }
+
```

Comment:
From looking at the Florence code, I don't think you need the RSS feed response? 

Ah but I can see it's a dependency in the other code. Fair enough.